elements/pfe-band/src/pfe-band.scss

```diff
@@ -103,246 +183,228 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
-          grid-template-areas:
-            "header header"
-            "body aside"
-            "footer footer";
-        }
-        :host([has_header][has_footer][has_aside][aside-height="full"]) & {
-          grid-template-areas:
-            "header aside"
-            "body aside"
-            "footer aside" ;
-        }
-        :host([has_header][has_footer][has_aside][aside-desktop="left"]) & {
-          grid-template-areas:
-            "header header"
-            "aside body"
-            "footer footer";
-        }
-        :host([has_header][has_footer][has_aside][aside-desktop="left"][aside-height="full"]) & {
-          grid-template-areas:
-            "aside header"
-            "aside body"
-            "aside footer";
-        }
-      }
+// For each section in the band layout,
+// define their internal behavior for slotted elements
+@include browser-query(ie11) {
+  .pfe-band__container {
+    @include pfe-clearfix;
+  }
+  .pfe-band__header {
+    float: left;
+  
+    :host([aside-mobile="top"]) & {
+      float: right;
+    }
+
+    :host(:not([aside-height="full"])) & {
+      width: 60%;
     }
   }
 
-  // For each section in the band layout
-  @each $section in (header, body, aside, footer) {
-    &__#{$section} { // i.e., pfe-band__header
-      margin-bottom: pfe-local(gutter--vertical);
-      // Apply the standard grid
-      @supports (display: grid) {
-        display: grid;
-        row-gap: pfe-local(gutter--vertical);
-        column-gap: pfe-local(gutter--horizontal);
-        margin-bottom: 0;
-        // Apply the section as a named grid-area
-        grid-area: $section;
-        // Allow users to hook into the layout with custom columns
-        grid-template-columns: pfe-local(layout, $region: #{$section});
-        grid-template-rows:    auto;
-      }
+  .pfe-band__body {
+    float: left;
+
+    :host([aside-mobile="top"]) & {
+      float: right;
+    }
+  
+    :host(:not([aside-height="full"])) & {
+      width: 60%;
     }
   }
 
-  &__aside {
+  .pfe-band__footer {
+    float: left;
+    clear: both;
+
+    :host([aside-mobile="top"]) & {
+      float: right;
+    }
+  
+    :host(:not([aside-height="full"])) & {
+      width: 60%;
+    }
+  }
+
+  .pfe-band__aside {
+    flex-grow: 1;
     align-self: flex-start;
+
+    float: right;
+    
+    width: 35%;
+    margin: 0 2.5%;
+      
+    :host([aside-desktop="left"]) & {
+      float: left;
+    }
   }
 
-  @include browser-query(ie11) {
-    &__body {
-        width: 60%;
-        float: left;
+  .pfe-band__wrapper {
+    width: 60%;
+    float: left;
+          
+    :host([aside-desktop="left"]) & {
+      float: right;
     }
-    &__aside {
-      float: left;
-      width: 35%;
-      margin: 0 2.5%;
+  }
+}
+
+.pfe-band__aside {
+  flex-grow: 1;
+  align-self: flex-start;
+}
+
+@each $section in (header, body, footer, aside) {
+  .pfe-band__#{$section} {
+    // Default margins if grid is not used
+    &:not(:last-child) {
```

Comment:
I'm validating if the class `.pfe-band__header` (etc.) is the last band in the container.  It might just need the margin removed if grid layout is used.  Let me check.

Reply:
Done and fixed